src/Controls/src/Core/SwipeView/SwipeView.cs

```diff
@@ -115,18 +117,12 @@ void SwipeItemsPropertyChanged(object sender, PropertyChangedEventArgs e)
 			{
 				if (sender is SwipeItems swipeItems)
 					SendChange(swipeItems);
-
-				if (sender is IEnumerable<ISwipeItem> enumerable)
```

Comment:
I don't quite understand this check. All of the SwipeItems are strongly typed to `SwipeItems` so you aren't going to get an `IEnumerable<ISwipeItem>` right?